segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -195,6 +199,9 @@ public void setupAppend(SetupAppend setupAppend) {
                                     // Last event number stored according to Segment store.
                                     long eventNumber = attributes.getOrDefault(writerAttributeId, Attributes.NULL_ATTRIBUTE_VALUE);
 
+                                    String indexSegment = getIndexSegmentName(newSegment);
+                                    populateEventSizeForIndexSegmentAppends(writer, indexSegment, setupAppend.getRequestId());
```

Comment:
What size is to be compared for Index storage?
As Index segment appends will be of fixed size based on Object AppendIndex .This has to be clarified.

Reply:
Currently we are using a random constant, but this needs to be finalized.